src/app/server/CommissioningWindowManager.cpp

```diff
@@ -242,6 +242,49 @@ void CommissioningWindowManager::OnSessionEstablished(const SessionHandle & sess
     }
 }
 
+// Function called when NFC-based commissioning has been started.
+// Advertizing can be stopped.
+// FailSafe timer is started
+// PASE session can be expired
+void CommissioningWindowManager::OnNfcBasedCommissioningStarting()
+{
+    ChipLogProgress(AppServer, "OnNfcBasedCommissioningStarting");
+
+    DeviceLayer::SystemLayer().CancelTimer(HandleSessionEstablishmentTimeout, this);
+
+    if (mAppDelegate != nullptr)
+    {
+        mAppDelegate->OnCommissioningSessionStarted();
```

Comment:
So normally that indicates PASE was established.... but clearly it means something else here?  I would have expected at least documentation changes to the app delegate.